BCI_analysis/plot/plot_imaging.py

```diff
@@ -263,3 +264,95 @@ def plot_trialwise_closed_loop(dataDict, dayOfInterest):
     plt.show()
 
 
+def correlation_deltaActivity(dataDict, dayOfInterest):
+    ## Set Up Day of Session
+    # dayOfInterest = input('Day of interest (day 1 = -1... dont do day 1): ')
+    if int(dayOfInterest) == 0:
+        while int(dayOfInterest) <= 0: #this is so we dont get stupid values
+            print('Incorrect Day Entry, redo please')
+            dayOfInterest = input('Day of interest (Day 1 = "1") ')
```

Comment:
using input() in a script is generally not what we want to do since we want the script to run without any supervision. If we run the script in the background or on a cluster, we generally have no way to give an input. If this is a part of the pipeline, we want this to run on 100 different sessions, its upto the one writing the script to use the functions appropriately. So if someone does give stupid values, its their fault :)